src/derived/openmetrics.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2024, Red Hat.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+# 
+# This file is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+#
+# Openmetrics derived metrics
```

Comment:
"camel case" is used everywhere else when describing "OpenMetrics", can we make this match?